sdk-core/src/main/java/care/data4life/sdk/migration/MigrationContract.kt

```diff
@@ -16,34 +16,34 @@
 
 package care.data4life.sdk.migration
 
-import care.data4life.sdk.network.model.NetworkModelContract
+import care.data4life.sdk.network.NetworkingContract
 import care.data4life.sdk.tag.Annotations
 import care.data4life.sdk.tag.Tags
-import io.reactivex.Observable
-import io.reactivex.Single
 
 @Retention(AnnotationRetention.SOURCE)
 @MustBeDocumented
 annotation class Migration(val message: String)
 
-interface MigrationContract {
+class MigrationContract {
```

Comment:
I would split internal and external contracts into seperate interface files. This ensures that a contract can't be instantiated and it clearly seperates internal from external APIs

Reply:
As discussed somewhere else. Yeah we should do that, but I like to make it as a follow up, since we have other places which also needs to be aligned.

Reply:
Sure 👍